frontend/src/components/TablePreview.tsx

```diff
@@ -37,7 +37,7 @@ const TablePreview = (props: Props) => {
                 minWidth: 150,
                 Cell: (props: any) => {
                   const row = props.row.original;
-                  return row[header] || null;
+                  return row[header].toLocaleString() || null;
```

Comment:
Should we check for null for row[header] before calling toLocaleString()

Reply:
I didn't run into any issues when testing, but it can't hurt.